.travis.yml

```diff
@@ -33,3 +34,6 @@ env:
     - secure: GVYDRNMwVypUDJXUQe9SCobwVCezyPODG4T2sMFZdZ3IjBV3vpPQxG0vKj5LL3HjRULb6D8X60Rsso/EQKSVs6cbRhU3h1Am4tgO/KjUswTcf97RJMk9a+pz+KmEFemJXYatpqK5uusYaJcMoKj5D4FeR2Z1FQjgLVV5y6dqAZc+lBzGq2l3QJbBgdwgIb7PLPBszdajpO5LZ3rcxKm+VvvittFqCCNA6V8NINnByByYpxy+BRm2JGrbOiDWzNC41RVbOY/xsRkOAHFKKJofh3sSHf/YAVyavCGi0nwmBevsoGS9Lmt9wGPydvuW7mQ58QODxjlpKoidohQRg81TY/rutl+Ul0a3xVFmTNVwK5dqfBekfwZBI7esBat864LdoWXViH93eQHbkYANUMMb14z5TBWprmAwRdUAUhVXxI201bYFpHTRl8ejBvtT/x0B38CRYd3cM2qjKNkhuOq3G2bCcU1iSW5Kn8U8M+sKI8rXkNtzqSKlQzPXznFhmVybNUh8K6FysIaPDwdtkfnzq9Obw8p1fLcHBts1BeaPxzF28GnTV/nsvTV44VF1sLJJ9kep7YlGzNfNM/eTianlQ1TZozVp+z4sAuytRkYoNgC+NpMnnwdqUk74AvKw2ExRHZbMLtMlKvcliV9etzdiaC0ZbKIFCjS1QFOFB2mpDaI=
     - secure: cq/0b2mHyfkpNtoecmJBhsq37vySkzeaH0TN06jAtW9/zGkm5yaIJkPUCdlJmW0on/xgXLCTvrx17nuRXQJmNpO32KfUOPfn7VIKS054Y8s86z61dKarWthTruNEosIdnmp88lV+V9lsjmAlYZv23u0yMDx4U34FYumPrH14eb27gGd/Xc7+LZpMYf3okxGPRtdxZ5KRO05v9dHcR3IthITixL26M6Z2VUb79fXkMiQ5Z17zh8sh1elS6AnE8SkcLYAefHhdJP18vFx3dmMlYY7m+qwfUmZ3YCZjUUYxA1Z+dnH5/mRNww4Se47TZFmoyHycq+P29znH+P7fSN5TnzTd6B5yPzIjFFTogv7BlPOgpq67J7vKeEyJ7mTYa5br84wnIIw1RJOltzhn2DO3YhXGysPzw8icz1RTlOASW86kUBAGiCzHW9iQGk2wll0ARohHSplahN8Qyb5vtUnF9Qep7fS6DTUHqAFCXiS1g7PEmvTD0ns5j+GNXLUy56LMrotpyV+zO5p8822Vu6315dbDkj7+znh+XTXG5we0AZIEAHK/F3BXy3URWmMdnrowPvlMMB20LLGvxOyaZMQsqBxYypHeKXqDMy9+M16LMJdUQRFCQFFunyGedHLJFyucs3ao6L0sxUdirXKFeoXnuB1QhLHQOphwhENuz1OLWmk=
     - secure: YWJ9ho4mwXjy4GrnbjehDlP5wFA6P8KLZMb+jOCEFNI9wKSiKzUUeIqGNog6Q9KCHE+RMEFz/kqE+7jNhhOgGGv0mRF0RVfHL0SkamhD0NMxiC1vITk3+oqMi/v1sK/V88WbfM4+1sAcFqV7yoptyWw/Db7U/MdLcRZaVe2zMz6s4P811CZjpT2Y/EkwxWcxejTCxptAvAkIfrgIyI28xW9hgA9oWvQJjD42ShKd445AC9qNq8E2cY8ukFIMik/FNHYWQMmdUTYOqX68O6fn2lPcIayPNCl0UuY6L9DB0KS0ZxrItX6/0JQU/CR8iFHWxshACvxLAMLUofPl/0QVkoSVaCGrKCX8WpjQ4K2FV/xiXa6xPXSFcI4RRsGkpXe3S6BHdWdH7whEsLDhvIJN3WQREWYqujXNi2imCO08+N0/GuDxfIcCzkrsEZYk5d1h8HQTk6Ml9ahhbL5EbrgEO6z8C/cLoSwebeYPeO7Wlw8y1f0pT8yIMH0cAI0PrdUzhQOgSgMUZGgntCynHyJymwtlHV7y829qsxvf3M0IpjCwK3VAdQLQKt2evFOT2hIbubpzvR6u26zeflpiMr/U8/uZnu3XYCPFLwkAZEDLb1LNfDiXOZY+cG+aX0R/m3hPDv5G5iH0OCTCwJ/usGSRMxdhwgQqltXrhGLUkvKcy8w=
+  matrix:
```

Comment:
I thought about that and added `if [[ $SCALA_VERSION == '2.11' ]]` condition that runs release only in one job, the other one is dummy.